ddp-workspace/projects/ddp-pancan/src/app/app.module.ts

```diff
@@ -155,6 +147,11 @@ export function translateFactory(translate: TranslateService,
             ],
             multi: true
         },
+        // Ensure that sorting of autocomplete picklist options is as specified
+        {
+            provide: PicklistSortingPolicy,
+            useValue: new PicklistSortingPolicy(SortOrder.ALPHABETICAL, 'UNKNOWN')
+        }
```

Comment:
The default policy leave things exactly as they are. This makes things alphabetical and pushes the UNKNOWN option to last.